Sources/EdgeNetworkHandlers/RequestContextData.swift

```diff
@@ -10,10 +10,34 @@
 // governing permissions and limitations under the License.
 //
 
+import AEPServices
 import Foundation
 
 /// Property that holds the global XDM context data within an `EdgeRequest` object.
 /// It is contained within the `EdgeRequest` request property.
 struct RequestContextData: Encodable {
-    let identityMap: IdentityMap?
+    var xdmPayloads: [[String: AnyCodable]] = []
```

Comment:
I don't think we need the RequestContextData struct anymore, we can just use 
```
/// struct EdgeRequest
let xdm: [String: AnyCodable]?
```

Reply:
I'd prefer to keep the struct as-is since it has custom encoding. It makes testing easy, and we don't need to have a custom encoding for the entire request.

Reply:
But with your changes this struct is no longer used for other custom fields, the EdgeRequest doesn't have ContextData, but rather meta, xdm (which represents the xdm payloads applied for the entire request) and events.